ui/app/contexts/metametrics.new.js

```diff
@@ -117,20 +147,16 @@ export function MetaMetricsProvider ({ children }) {
           properties: {
             ...omit(properties, ['revenue', 'currency', 'value']),
             category,
-            exclude_meta_metrics_id: excludeMetaMetricsId,
-          },
-          context: {
-            version: global.platform.getVersion(),
-            environment: process.env.METAMASK_ENVIRONMENT,
-            platform: getPlatform(),
             network,
-            environment_type: getEnvironmentType(),
+            exclude_meta_metrics_id: excludeMetaMetricsId,
```

Comment:
Is this useful to track? I don't think we tracked it before - we just used this to determine whether to anonymize the event

Reply:
In retrospect, no -- you get the same information by seeing if an anonymousId or userId was provided.